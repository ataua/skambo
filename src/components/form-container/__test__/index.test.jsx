import React from 'react';
import FormContainer from '../index';
import TestRenderer from 'react-test-renderer';

describe('Form  Snapshot test', () => {
  it('Form  Snapshot test Render', () => {
    const tree = TestRenderer.create(<FormContainer />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
