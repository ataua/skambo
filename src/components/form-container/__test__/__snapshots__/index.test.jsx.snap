// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Form  Snapshot test Form  Snapshot test Render 1`] = `
<div
  className="sc-bdnylx leTbsp"
>
  <div
    className="sc-gtssRu brhlVN"
  >
    <div
      className="sc-hKFyIo dJqfuM"
    >
      <img
        alt="Logotipo Skambo"
        src="register-icon.svg"
      />
    </div>
    <div
      className="sc-dlnjPT cOIIvD"
    />
  </div>
</div>
`;
